ecl/hql/hqlgram.y

```diff
@@ -3505,13 +3521,9 @@ outputFlag
                             $$.setExpr(createExprAttribute(csvAtom, args));
                             $$.setPosition($1);
                         }
-    | COMPRESSED        {
-                            $$.setExpr(createAttribute(compressedAtom));
-                            $$.setPosition($1);
-                        }
-    | __COMPRESSED__    {
-                            $$.setExpr(createAttribute(__compressed__Atom));
-                            $$.setPosition($1);
+    | compressionOptions
+                        {
```

Comment:
production is better removed - i.e. remove { ... }

Reply:
will remove